storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -6,13 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} provides automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers.
-
-:FeatureName: CSI automatic migration
-include::snippets/technology-preview.adoc[leveloffset=+1]
+In-tree storage drivers that are traditionally shipped with {product-title} are being deprecated and replaced by their equivalent Container Storage Interface (CSI) drivers. {product-title} provides automatic migration for certain supported in-tree volume plug-ins to their equivalent CSI drivers.
 
 include::modules/persistent-storage-csi-migration-overview.adoc[leveloffset=+1]
 
+=== Additional resources
+* xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-automatic-ga_persistent-storage-csi-migration[Automatic migration of in-tree volumes to CSI]
+
+* xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-enable_persistent-storage-csi-migration[Manually enabling CSI automatic migration]
+
+include::modules/persistent-storage-csi-migration-automatic-ga.adoc[leveloffset=+1]
+
 include::modules/persistent-storage-csi-migration-enable.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
```

Comment:
Git won't let me comment on the line below, but it should also be a `.Additional resources` style heading (it's tacking onto the preceding module, not an additional resource link for the entire assembly)